database/migrations/2017_04_27_141550_create_voucher_submission_pivot_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateVoucherSubmissionPivotTable extends Migration
```

Comment:
Submission is for one Trader the User adminsters.
A User records many Vouchers on a Sumbssion
Submissions have state (see above)
Vouchers MAY appear on many submissions BUT only sumbissions that are "accepted" can allocate thier associated Trader with the voucher. subsequent submissions will bounce it as "collected" by a Trader. 

attempts to cope with edge cases in collection i suppose.